txtorcon/torconfig.py

```diff
@@ -423,6 +424,11 @@ class EphemeralHiddenService(object):
     https://gitweb.torproject.org/torspec.git/tree/control-spec.txt#n1295
     '''
 
+    @classmethod
+    def _is_valid_keyblob(cls, key_blob_or_type):
+        return (isinstance(key_blob_or_type, str) and
+                re.match(r'[^ :]+:[^ :]+$', key_blob_or_type))
```

Comment:
Oh man, you mentioned that and I totally forgot! Sorry :/ I agree, I think it should be `six.text_type`.

Would you like me to fix that? Will it break stuff because people would then have to use `u'ALG:KEY'`?